docker-compose.yaml

```diff
@@ -1,9 +1,10 @@
 services:
   status:
-    # "ext-" for the Hugo extended edition including git (https://github.com/klakegg/docker-hugo/blob/master/README.md#hugo-extended-edition)
-    image: klakegg/hugo:0.95.0-ext-asciidoctor
+    # "ext-" for the Hugo extended edition including git (https://github.com/floryn90/docker-hugo/blob/master/README.md#hugo-extended-edition)
+    user: ${RUN_AS_USER}
+    image: floryn90/hugo:0.121.1-ext-asciidoctor
```

Comment:
Alas there are no `linux/arm64` for this image's tag: https://hub.docker.com/layers/floryn90/hugo/ext-asciidoctor-onbuild/images/sha256-222d38bf30305cd91eb9badee59b7156ac6fff14cb331393993f463155eda0c8?context=explore

which excludes any macOS user